wasmtime/_value.py

```diff
@@ -1,10 +1,11 @@
 from ._ffi import *
 from wasmtime import ValType
+import typing
 
 
 class Val:
     @classmethod
-    def i32(cls, val):
+    def i32(cls, val: int) -> "Val":
```

Comment:
This seems like a pretty common pattern we have, where within a `class` definition there's methods that take/return that class. Is there really no other way to express this than with a string?

Reply:
The quotes are there because `Val` isn't yet known to Python as it's compiling the definition of the class. It's a [forward-reference](https://www.python.org/dev/peps/pep-0484/#id36).

Reply:
That makes sense I'm just surprised that such a common pattern doesn't have better support...